tests/rkt_run_pod_manifest_test.go

```diff
@@ -504,13 +518,12 @@ func TestPodManifest(t *testing.T) {
 			continue
 		}
 
-		j := 0
-		for name, patches := range tt.images {
-			imageFile := patchTestACI(name, patches...)
+		for j, v := range tt.images {
+			imageFile := patchTestACI(v.name, v.patches...)
```

Comment:
Yeah, the second aci will overwrite the first one, and the second `os.Remove` will fail, but it's ok.